src/natcap/invest/coastal_vulnerability.py

```diff
@@ -2891,7 +2889,7 @@ def _validate_habitat_table_paths(habitat_table_path):
     Raises:
         ValueError if any vector in the ``path`` column cannot be opened.
     """
-    habitat_dataframe = pandas.read_csv(habitat_table_path, header=0)
+    habitat_dataframe = utils.read_csv_to_dataframe(habitat_table_path)

```

Comment:
Do we lose anything here by removing the `header=0` kwarg?

Reply:
I'm not sure why it was there, I don't believe it's doing anything since no column names are being passed. It should be equivalent to the default behavior. see pandas docs: [https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_csv.html](url)

Reply:
Interesting!  This does look a bit redundant to have here, then.